apis/meta/v1alpha1/converters.go

```diff
@@ -0,0 +1,51 @@
+package v1alpha1
+
+import (
+	"unsafe"
+
+	"github.com/sensu/sensu-go/internal/apis/meta"
+)
+
+// ConvertTo converts a *ObjectMeta to a *meta.ObjectMeta.
+// It panics if the to parameter is not a *meta.ObjectMeta.
+func (r *ObjectMeta) ConvertTo(to interface{}) {
+	ptr := to.(*meta.ObjectMeta)
+	convert_ObjectMeta_To_meta_ObjectMeta(r, ptr)
+}
+
+var convert_ObjectMeta_To_meta_ObjectMeta = func(from *ObjectMeta, to *meta.ObjectMeta) {
+	*to = *(*meta.ObjectMeta)(unsafe.Pointer(from))
+}
+
+// ConvertFrom converts the receiver to a *meta.ObjectMeta.
```

Comment:
Should this read "converts the receiver to a *ObjectMeta"?

Reply:
No, because the receiver is already a *ObjectMeta. I can try to rearrange the wording if it's confusing!

Reply:
What's confusing me is that it says that it converts the receiver, a `*ObjectMeta`, to a `*meta.ObjectMeta`. That's confusing because:
- it sounds like it's what `ConvertTo()` is doing
- it literally calls a function called `convert_meta_ObjectMeta_To_ObjectMeta`

Am I missing something here?